src/passes/LogExecution.cpp

```diff
@@ -39,9 +39,17 @@ namespace wasm {
 Name LOGGER("log_execution");
 
 struct LogExecution : public WalkerPass<PostWalker<LogExecution>> {
+  IString logger_module;
```

Comment:
```suggestion
  // The module name the logger function is imported from.
  Name loggerModule;
```

Reply:
done

Reply:
Aside from the comment, the diff also renamed the variable. `loggerModule` is more consistent with our naming conventions (compared to `logger_module`).

Reply:
oops, i haven't noticed it. done.